src/styles/components/_home.scss

```diff
@@ -19,59 +19,63 @@ $breakpoint-top-section-scale-up: 70rem;
     color: var(--foreground-color);
     max-width: 100rem; // 1600px/16
 
-    @media (min-width: $breakpoint-top-section-margin) {
-      margin: auto;
-    }
-
-    @media (min-width: $breakpoint-large-screen-860) {
+    @media (min-width: $breakpoint-scale-up) {
       padding: 0 1.25rem 1.25rem 8.5625rem; // top&bottom: 0; right: 20px/16; left: 137px/16
       display: flex;
       flex-basis: 100%;
       justify-content: space-between;
       margin-top: 2rem;
     }
-  }
 
-  &-top__text-column {
-    @media (min-width: $breakpoint-large-screen-860) {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      flex-basis: 45%;
-      padding-right: 1.875rem; // 30px/16
+    @media (min-width: $breakpoint-top-section-margin) {
+      margin: auto;
     }
 
-    h1 {
-      font-size: 1.5625rem; // 25px/16
-      font-weight: 400;
-      line-height: 2.125rem; // 34px/16
-
-      @media (min-width: $breakpoint-large-screen-860) {
-        font-size: map.get($font-sizes, "md");
-        margin: 0;
+    &__text-column {
+      @media (min-width: $breakpoint-scale-up) {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        flex-basis: 45%;
+        padding-right: 1.875rem; // 30px/16
       }
 
-      @media (min-width: $breakpoint-top-section-scale-up) {
-        font-size: map.get($font-sizes, "xl");
-        margin: 0;
-        line-height: 4.1875rem; // 67px/16
+      h1 {
+        font-size: 1.5625rem; // 25px/16
+        font-weight: 400;
+        line-height: 2.125rem; // 34px/16
+
+        @media (min-width: $breakpoint-scale-up) {
+          font-size: map.get($font-sizes, "md");
+          margin: 0;
+        }
+
+        @media (min-width: $breakpoint-grid-large-screen-1115) {
+          font-size: map.get($font-sizes, "xl");
+          margin: 0;
+          line-height: 4.1875rem; // 67px/16
```

Comment:
same comment

Reply:
It could be nice to add a line-height map with something like

```css
default: 1,
compact: 1.1,
expanded: 1.4,
```

Reply:
I think you're probably right. However, for this card, I think doing a new line-height map might be out of scope. I'll definitely create a new card though!

Reply:
Fixed for this file too: https://github.com/sparkbox/accessible-components/pull/89/commits/5caa631fbe674d5c7322e967e502754ce8687642